hyrax-active_encode.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |s|
   s.add_dependency "hyrax", "~> 2.1"
 
   s.add_development_dependency 'bixby'
+  s.add_development_dependency 'coffee-rails'
```

Comment:
Was needed for some reason during testing.  It might not be necessary for this PR at the moment but might be needed for the other `output_service` branch.  I think we should just leave it in for now and try removing it after everything is in master.